packages/Chem/package.js

```diff
@@ -141,6 +141,42 @@ class ChemPackage extends DG.Package {
     }
   }
 
+  //name: chemSimilaritySpeImpl
+  //input: column fingerprints [fingerprints]
+  //input: int dimension
+  //output: dataframe result
+  async chemSimilaritySpeImpl(fingerprints, dimension) {
+    var core = _fingerprintSimilarity;
+    var coordinates = stochasticProximityEmbedding(fingerprints.length, (x, y) => 1.0 - core(fingerprints.get(x), fingerprints.get(y)),
```

Comment:
Where are we passing the number of dimensions?

Reply:
This is an old commit, pls check the recent ones. I already removed this function.